hermes-common/src/main/java/pl/allegro/tech/hermes/common/hook/HooksHandler.java

```diff
@@ -23,7 +23,8 @@ public void addShutdownHook(ServiceAwareHook hook) {
     public void shutdown(ServiceLocator serviceLocator) {
         try {
             shutdownHooks.forEach(c -> c.accept(serviceLocator));
-        } finally {
+        }
```

Comment:
\[Checkstyle\] ERROR: '\}' at column 9 should be on the same line as the next part of a multi\-block statement\.